Makefile

```diff
@@ -410,3 +441,14 @@ go_docs: check_godoc ## Generate documentation for the project
 	echo "Visit http://localhost:6060/pkg/pocket/"
 	godoc -http=:6060
 
+.PHONY: ignite_openapi_gen
+ignite_openapi_gen: ## Generate the OpenAPI spec for the Ignite API
```

Comment:
Wdyt about just `openapi_gen` as this is (rightly so IMHO) under the "documentation" section, as opposed to the "ignite helpers" section? :thinking: